Source/WebCore/page/scrolling/ScrollingCoordinatorTypes.h

```diff
@@ -89,6 +89,8 @@ struct ScrollableAreaParameters {
 
     bool useDarkAppearanceForScrollbars { false };
 
+    ScrollbarWidth scrollbarWidthStyle { ScrollbarWidth::Auto };
```

Comment:
If I change this to ScrollbarWidth::Thin it renders as thin. But that seems odd given it's definitely set inside of AsyncScrollingCoordinator. I assume I'm missing so code that handles the value being updated.